src/lib/strings/embed-player.ts

```diff
@@ -451,6 +455,32 @@ export function parseEmbedPlayerFromUrl(
         return undefined
     }
   }
+
+  const bandcampRegex = /^[a-z\d][a-z\d-]{2,}[a-z\d]\.bandcamp\.com$/i
```

Comment:
This won't support custom domains, do we care? I don't think there would be a way to support custom domains without prefetch

Reply:
Good point. I think this is a good start. There was another issue that mentioned beefing up the prefetch. They must do something when you add a link because they fetch and store a thumbnail image, so presumably they could grab the embed code from the meta tags and store it. But that’s beyond what we can change here, and also somewhat client-dependent...